code/stylesheet.dm

```diff
@@ -75,6 +75,10 @@ h1.alert, h2.alert		{color: #000000;}
 .grey					{color: #585858; font-family: Dotum, sans-serif;}
 .skeleton				{color: #585858; font-weight: bold; font-style: italic;}
 .gutter					{color: #61380B; font-style: italic;}
+.orange					{color: #ffa500;}
+.orangei				{color: #ffa500; font-style: italic;}
```

Comment:
There's also `bold` and `italic` classes si you're better off using `<span class='orange italic'> instead anf removing these.